packages/cli/src/commands/sandbox/sandbox_command.ts

```diff
@@ -201,4 +193,16 @@ export class SandboxCommand
         await this.sandboxFactory.getInstance()
       ).delete({ name: this.sandboxName });
   };
+
+  validateDirectory = (option: string, dir: string) => {
+    let stats;
+    try {
+      stats = fs.statSync(dir, {});
```

Comment:
it seems that `check` might be supporting async functions.
Can you give it a try and convert this to async?

Ref: https://github.com/yargs/yargs/pull/1872

Reply:
Was not aware of this, good to know and done in [4cb5b72](https://github.com/aws-amplify/amplify-backend/pull/643/commits/4cb5b72ca62b0f2ee5219b5f4155284602c46de4).

Reply:
any file system access if possible should be made async so that event loop is not blocked.